index.Rmd

```diff
@@ -97,7 +94,7 @@ Once importing is complete, you can go to the appliance settings and add a share
 If you like, you can install a Linux distribution on your own laptops or external hard drives. Compared to virtualisation, it is much faster to run a Linux distribution directly. -->
 ```
 
-Once you have everything ready, login into your Linux VM, try out RStudio/RKWard, and also open QGIS. Explore the environment a little to get used to it.
```

Comment:
It's a good idea to also check out RKWard, it should be installed.

Reply:
Yes, that's just because it ships two startup files, but it's not a problem.

Reply:
You should still keep the mention of RKWard there, that was the point of this comment :)